lib/floe/workflow/payload_template.rb

```diff
@@ -4,35 +4,73 @@ module Floe
   class Workflow
     class PayloadTemplate
       def initialize(payload)
-        @payload = payload
+        @payload          = payload
```

Comment:
Hard to tell from this PR, but do we even need `@payload` anymore,  or is just having `@payload_template` sufficient?

Reply:
It isn't used, we could remove it I just had it for consistency with the other classes which keep the raw payload

Reply:
Removed

Reply:
Yay. I want to remove payload (and workflow) from everywhere. But more of a side thought and don't have any good push behind it